uefi_app/SbsaAvsMain.c

```diff
@@ -451,6 +455,18 @@ ShellAppMainsbsa (
       }
   }
 
+	// Options with Values
+  CmdLineArg  = ShellCommandLineGetValue (ParamPackage, L"-t");
+  if (CmdLineArg != NULL) {
+    g_single_test = StrDecimalToUintn(CmdLineArg);
+  }
+
+  // Options with Values
```

Comment:
Define the new parameters at Line 44 to avoid build failure.
UINT32  g_single_test;
UINT32  g_single_module;

Reply:
Fixed. Created a fresh environment in a new Debian install to confirm there will be no more build issues.